pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1254,6 +1254,10 @@ func (og *operationGenerator) GenerateExpandVolumeFunc(
 		return volumetypes.GeneratedOperations{}, fmt.Errorf("Error finding plugin for expanding volume: %q with error %v", pvcWithResizeRequest.QualifiedName(), err)
 	}
 
+	if volumePlugin == nil {
```

Comment:
Modify `FindExpandablePluginBySpec` to return `nil, err` instead of current `nil, nil`?

I see `FindExpandablePluginBySpec` is only used in two places.

Reply:
No, I don't think it's an error. Especially for FlexVolume, in general, we should switch on "expand", but for specific volume driver used by FlexVolume, it may not be expandable. So I think it's better to deal with the nil situation here.